HHS/Head-Start-TTADP
# TTAHUB-446: Display Unique Program and Grant Specialists

## Description of change

1. The grantee search result should now show all Program Specialists (distinct and sorted comma separated list).
2. The grantee summary now shows all Program and Grant Specialists (distinct and sorted).

## How to test

Search for a grantee with Program Specialists. You should now see the list of Program Specialists. Click on a Grantee that has both Program and Grant Specialists. The summary page should show both listed.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-446


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
